services/management/ia/templates_service.go

```diff
@@ -31,6 +31,7 @@ import (
 
 	"github.com/golang/protobuf/ptypes"
 	"github.com/percona-platform/saas/pkg/alert"
+	saas "github.com/percona-platform/saas/pkg/alert"
```

Comment:
This is the same package as the one above :) 6798fff6c4e58a947ff291b9bd5f438e520eab02